scenarios/generate-synthetic-data/simulate-adversarial-interactions/askwiki/simulate_and_evaluate_ask_wiki.ipynb

```diff
@@ -0,0 +1,341 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Adversarial Simulator for a custom application - askwiki\n",
+    "\n",
+    "## Objective\n",
+    "\n",
+    "This tutorial provides a step-by-step guide on how to leverage adversarial simulator to simulate an adversarial question answering scenario against a custom application - askwiki.\n",
+    "\n",
+    "This tutorial uses the following Azure AI services:\n",
+    "\n",
+    "- promptflow-evals\n",
```

Comment:
Consider adding a link to relevant documentation

Reply:
Added